{% raw %}diff -rNu treebeard-4.0.1/admin.py treebeard-4.0.2a0/admin.py
--- treebeard-4.0.1/admin.py	2016-09-22 14:23:11.339782714 +0200
+++ treebeard-4.0.2a0/admin.py	2016-09-22 11:47:54.000000000 +0200
@@ -3,7 +3,7 @@
 import sys

 from django.conf import settings
-from django.conf.urls import patterns, url
+from django.conf.urls import url

 from django.contrib import admin, messages
 from django.http import HttpResponse, HttpResponseBadRequest
@@ -54,13 +54,13 @@
         """
         Adds a url to move nodes to this admin
         """
+        from django.views.i18n import javascript_catalog
+
         urls = super(TreeAdmin, self).get_urls()
-        new_urls = patterns(
-            '',
-            url('^move/$', self.admin_site.admin_view(self.move_node), ),
-            url(r'^jsi18n/$', 'django.views.i18n.javascript_catalog',
-                {'packages': ('treebeard',)}),
-        )
+        new_urls = [
+            url(r'^move/$', self.admin_site.admin_view(self.move_node), ),
+            url(r'^jsi18n/$', javascript_catalog, {'packages': ('treebeard',)}),
+        ]
         return new_urls + urls

     def get_node(self, node_id):
diff -rNu treebeard-4.0.1/templates/admin/tree_change_list_results.html treebeard-4.0.2a0/templates/admin/tree_change_list_results.html
--- treebeard-4.0.1/templates/admin/tree_change_list_results.html	2016-09-22 14:23:11.343782901 +0200
+++ treebeard-4.0.2a0/templates/admin/tree_change_list_results.html	2016-09-22 14:22:38.287782518 +0200
@@ -1,4 +1,3 @@
-{% load cycle from future  %}
 {% if result_hidden_fields %}
     <div class="hiddenfields"> {# DIV for HTML validation #}
         {% for item in result_hidden_fields %}{{ item }}{% endfor %}
diff -rNu treebeard-4.0.1/templates/admin/tree_list_results.html treebeard-4.0.2a0/templates/admin/tree_list_results.html
--- treebeard-4.0.1/templates/admin/tree_list_results.html	2016-09-22 14:23:11.343782901 +0200
+++ treebeard-4.0.2a0/templates/admin/tree_list_results.html	2016-09-22 14:22:42.943782562 +0200
@@ -1,4 +1,3 @@
-{% load cycle from future  %}
 {% if results %}
     <ul>
         {% for result in results %}{% endraw %}
